SQLStuff/setup.sql

```diff
@@ -0,0 +1,104 @@
+DROP TABLE IF EXISTS websites CASCADE;
+DROP TABLE IF EXISTS watchables CASCADE;
+DROP TABLE IF EXISTS watchables_sites CASCADE;
+DROP TABLE IF EXISTS organizations CASCADE;
+DROP TABLE IF EXISTS contributors CASCADE;
+DROP TABLE IF EXISTS projects CASCADE;
+DROP TABLE IF EXISTS download_mirrors CASCADE;
+DROP TABLE IF EXISTS org_members CASCADE;
+DROP TABLE IF EXISTS project_contributors CASCADE;
+DROP TABLE IF EXISTS licenses CASCADE;
+DROP TABLE IF EXISTS users CASCADE;
+DROP TABLE IF EXISTS user_watched_items CASCADE;
+DROP TABLE IF EXISTS project_licenses CASCADE;
+
+CREATE TABLE websites (
+	id	 serial NOT NULL,
+	name	varchar(40) NOT NULL,
+	uri	varchar(250) NOT NULL,
+	descr	varchar(400),
+	PRIMARY KEY(id),
+	UNIQUE (uri)
+);
+
+CREATE TABLE watchables (
+	id	serial NOT NULL,
+	PRIMARY KEY(id)
+);
+
+CREATE TABLE watchables_sites (
+	watchable_id	int NOT NULL REFERENCES watchables(id),
+	site_id		int NOT NULL REFERENCES websites(id)
+);
+
+CREATE TABLE organizations (
+	id	int NOT NULL REFERENCES watchables(id),
+	name	varchar(40) NOT NULL,
+	email	varchar(40) NOT NULL,
+	PRIMARY KEY(id)
+);
+
+CREATE TABLE contributors (
+	id	int NOT NULL REFERENCES watchables(id),
+	name	varchar(40) NOT NULL,
+	email	varchar(40) NOT NULL,
+	PRIMARY KEY(id)
+);
+
+CREATE TABLE licenses (
+	id		serial NOT NULL,
+	name		varchar(40) NOT NULL,
+	text_link	int NOT NULL REFERENCES websites(id),
+	UNIQUE(name, text_link),
```

Comment:
Licenses should really just be unique for name (since we don't want to have more than one link per license.)

Actually, we should consider just including the URI directly, as we don't gain anything from the extra information associated with a website entity.

Reply:
I'll change the unique to name.  We gain the ability to watch licenses with the foreign key.